CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.6.0 - 2016-03-30
+
+* Supported configuration of `specs` for certain browsers (see [#9]).
+* Added option `--grep` for running tests (see [#15]).
+* Removed russian documentation and added improved english one.
+* Fixed the bug in debug mode concerning with [q](https://github.com/kriskowal/q) module (see [#10]).
```

Comment:
Fixed the bug in debug mode concerning with -> Fixed crash when enabling debug mode